### PR TITLE
Beautify venue#show

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

**Comment by hennevogel** [comment]
This PR depends on #769

File: app/views/admin/venues/show.html.haml [hunk 1]
```diff
@@ -7,7 +7,12 @@
 -if @venue
   .row
     .col-md-6
-      =image_tag(@conference.venue.photo, class: 'img-responsive img-rounded') unless @conference.venue.photo.blank?
+      - if @conference.venue.location?
+        = render 'conference/venue_map'
+      - elsif @conference.venue.photo.blank?
+        %img{ "data-src": "holder.js/300x200?text=No Image Set", class: 'img-responsive img-rounded' }
```
Comment (differentreality) [inline app/views/admin/venues/show.html.haml]: `"data-src":` gives me an error, `"data-src" => `works

Comment (hennevogel) [inline app/views/admin/venues/show.html.haml]: What kind of error?

Comment (differentreality) [inline app/views/admin/venues/show.html.haml]: http://paste.opensuse.org/65196591

**Comment by ChrisBr** [comment]
@hennevogel please rebase!

**Comment by hennevogel** [comment]
Done

**Comment by differentreality** [comment]
:+1:

**Pull request merged**